hail/python/hailtop/aiocloud/aiogoogle/user_config.py

```diff
@@ -16,11 +16,12 @@
 def get_gcs_requester_pays_configuration(
         *,
         gcs_requester_pays_configuration: Optional[GCSRequesterPaysConfiguration] = None,
+        requester_pays_project_id: Optional[str] = None,
```

Comment:
Does this differ from the variant of `GCSRequesterPaysConfiguration` that is just `str`? I thought that `gcs_requester_pays_configuration = 'my-project'` meant "use `my-project` for all requester pays requests".

Reply:
I think I'm just going to do that. Thanks for the suggestion.